cloudbuild/cloudbuild.yaml

```diff
@@ -9,39 +9,39 @@ steps:
     id: 'gcr-push'
     args: ['push', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID']
 
-  # Run presubmit tests in parallel for 1.5 Debian image
-  - name: 'gcr.io/cloud-builders/kubectl'
-    id: 'dataproc-1.5-debian10-tests'
-    waitFor: ['gcr-push']
-    entrypoint: 'bash'
-    args: ['cloudbuild/run-presubmit-on-k8s.sh', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID', '$BUILD_ID', '1.5-debian10']
-    env:
-      - 'COMMIT_SHA=$COMMIT_SHA'
-      - 'CLOUDSDK_COMPUTE_REGION=us-central1'
-      - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
-
-  # Run presubmit tests in parallel for 1.5 Rocky Linux image
-  - name: 'gcr.io/cloud-builders/kubectl'
-    id: 'dataproc-1.5-rocky8-tests'
-    waitFor: ['gcr-push']
-    entrypoint: 'bash'
-    args: ['cloudbuild/run-presubmit-on-k8s.sh', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID', '$BUILD_ID', '1.5-rocky8']
-    env:
-      - 'COMMIT_SHA=$COMMIT_SHA'
-      - 'CLOUDSDK_COMPUTE_REGION=us-central1'
-      - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
-
-  # Run presubmit tests in parallel for 1.5 Ubuntu image
-  - name: 'gcr.io/cloud-builders/kubectl'
-    id: 'dataproc-1.5-ubuntu18-tests'
-    waitFor: ['gcr-push']
-    entrypoint: 'bash'
-    args: ['cloudbuild/run-presubmit-on-k8s.sh', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID', '$BUILD_ID', '1.5-ubuntu18']
-    env:
-      - 'COMMIT_SHA=$COMMIT_SHA'
-      - 'CLOUDSDK_COMPUTE_REGION=us-central1'
-      - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
-
+#  # Run presubmit tests in parallel for 1.5 Debian image
```

Comment:
same as the previous/next two comments.  leave comments for one PR and remove them in the next to reduce file size and execution time.